src/toil_scripts/adam_gatk_pipeline/align_and_call.py

```diff
@@ -136,8 +136,10 @@ def build_parser():
 
     parser = argparse.ArgumentParser()
 
+    in_production_mode = not mock_mode()
```

Comment:
I'd prefer inlining this.

```
in_production_mode
not mock_mode()
```

See how the inlined version is actually shorter?